### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by dhananjay0610** [speer-technologies/aircall#25]
yarn run v1.22.19
warning ../../package.json: No license field
$ webpack-dev-server --mode development --open
(node:60107) [DEP0111] DeprecationWarning: Access to process.binding('http_parser') is deprecated.
(Use `node --trace-deprecation ...` to show where the warning was created)
ℹ ｢wds｣: Project is running at http://localhost:8080/
ℹ ｢wds｣: webpack output is served from /
node:internal/crypto/hash:71
  this[kHandle] = new _Hash(algorithm, xofLen);
                  ^

Error: error:0308010C:digital envelope routines::unsupported

**Comment by subhajit033** [comment]
I don't think this will work because this react version is <18 , you have a create a fresh react app and copy the necessary file from repo
